gtsam/geometry/Cal3DS2.h

```diff
@@ -33,9 +33,13 @@ class GTSAM_EXPORT Cal3DS2 : public Cal3DS2_Base {
 
   typedef Cal3DS2_Base Base;
 
+private:
+
+  static const size_t dimension_ = 9;
```

Comment:
Why?

Reply:
This was a TODO and it made sense from a future-proofing perspective to make the variable static.

Reply:
Sorry, no, this change seems to *add* to the plethora of places where we define dimension. the `enum { dimension = 9 }` sees to already do the trick. Just use that in the functions.

Reply:
I am a bit confused. I am not sure how it adds, since we are simply replacing `enum { dimension = 9 }` with `enum {dimension = dimension_}`. Also, I believe the enum version doesn't compile, the reason for which I don't know off the top of my head.